frontend/src/employee-frontend/components/child-information/service-need/ServiceNeedRow.tsx

```diff
@@ -100,6 +100,7 @@ function ServiceNeedRow({ serviceNeed, onReload }: Props) {
         ) : (
           <LabelValueList
             spacing="large"
+            labelWidth="240px"
```

Comment:
Hmm I'll try that. Does fit-content(40%) mean fit content but max width 40%? I think 40% might be too much here now since the container width has increased.

Reply:
Yeah it's the maximum. 40% can be a lot, but it's not that much with smaller screens. The left side shouldn't be very long in any case anyway.